go.mod

```diff
@@ -1,70 +1,23 @@
 module github.com/livepeer/go-livepeer
 
-go 1.13
+go 1.12
 
 require (
 	contrib.go.opencensus.io/exporter/prometheus v0.1.0
-	github.com/allegro/bigcache v1.2.1 // indirect
-	github.com/apilayer/freegeoip v3.5.0+incompatible // indirect
-	github.com/aristanetworks/goarista v0.0.0-20190909155222-05df9ecbb0dc // indirect
-	github.com/aws/aws-sdk-go v1.23.19
-	github.com/btcsuite/btcd v0.0.0-20190824003749-130ea5bddde3 // indirect
+	github.com/aws/aws-sdk-go v1.25.48
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/cespare/cp v1.1.1 // indirect
-	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/docker/docker v1.13.1 // indirect
-	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/elastic/gosigar v0.10.5 // indirect
-	github.com/ethereum/go-ethereum v1.9.3
-	github.com/fatih/color v1.7.0 // indirect
-	// replace example.com/some/dependency => example.com/some/dependency-fork v1.2.3
-	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
-	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff // indirect
-	// github.com/gballet/go-libpcsclite v0.0.0-20190403181518-312b5175032f // indirect
+	github.com/ethereum/go-ethereum v1.9.9
```

Comment:
I'm nervous about this change; would prefer not to update dependencies if we can avoid it, especially things that we are heavily dependent on such as `go-ethereum`